server/container_create.go

```diff
@@ -393,6 +393,11 @@ func (s *Server) getAppArmorProfileName(annotations map[string]string, ctrName s
 	}
 
 	if profile == apparmor.ProfileRuntimeDefault {
+		// reload default apparmor profile if it is unloaded.
+		if s.appArmorProfile == apparmor.DefaultApparmorProfile {
+			apparmor.LoadDefaultAppArmorProfile()
```

Comment:
This shouldn't be here IMO. It should be where this is called -- it's odd that a function called "get" is loading profiles.